src/solc_0.8/polygon/root/land/LandTunnel.sol

```diff
@@ -69,6 +70,10 @@ contract LandTunnel is FxBaseRootTunnel, IERC721MandatoryTokenReceiver, ERC2771H
         _trustedForwarder = trustedForwarder;
     }
 
+    function togglePauseTransferToL2() external onlyOwner returns (bool) {
```

Comment:
I'd rather have 2 functions pause & unpause like this
https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/token/ERC1155/presets/ERC1155PresetMinterPauser.sol#L86